tools/gn

```diff
@@ -303,7 +303,7 @@ def to_gn_args(args):
 
   if gn_args['target_os'] == 'ios':
     gn_args['use_ios_simulator'] = args.simulator
-  else:
+  elif gn_args['target_os'] == 'mac':
```

Comment:
It's a bit more complicated since you can target Android while on a mac.

Reply:
But simulator is iOS specific right?